self-core-impl/src/main/java/com/selfxdsd/core/managers/StoredProjectManager.java

```diff
@@ -210,15 +209,12 @@ public double percentage() {
 
     @Override
     public BigDecimal commission(final BigDecimal value) {
-        final double twoDigitsPercentage = Double.valueOf(
-            new DecimalFormat("0.00").format(this.percentage)
```

Comment:
@charlesmike On my end the test for this part was failling with 
`java.lang.NumberFormatException: For input string: "8,00"` due to using comma separator imposed by `Locale`, most likely. So I've changed it to use `BigDecimal`s only.